tests/phpunit/MigrateTest.php

```diff
@@ -221,6 +215,32 @@ public function testMigrateSuccess(
         $migrate->setMigrationsClientFactory(fn() => $migrationsClientMock);
 
         $migrate->run();
+
+        if ($migrateSecrets) {
+            $records = array_filter(
+                $logsHandler->getRecords(),
+                fn(array $record) => in_array('secrets', $record['context'] ?? [], true)
+            );
+            self::assertCount(4, $records);
+
+            $record = array_shift($records);
+            self::assertSame('Migrating secrets in configurations', $record['message']);
```

Comment:
I tahle hláška v logu teda taky není úplně ideální - asi by to mělo být "migrating configurations with secrets") (taky klidně v nějakým dalším PR)

Reply:
Ok, nechám na další PR.